hail/src/main/scala/is/hail/expr/ir/functions/SetFunctions.scala

```diff
@@ -58,32 +58,32 @@ object SetFunctions extends RegistryFunctions {
           ToStream(Ref(x, TArray(t)))))
     }
 
-    registerIR2("intersection", TSet(tv("T")), TSet(tv("T")), TSet(tv("T"))) { (_, s1, s2) =>
+    registerIR2("intersection", TSet(tv("T")), TSet(tv("T")), TSet(tv("T"))) { (_, s1, s2, _) =>
       val t = s1.typ.asInstanceOf[TSet].elementType
       val x = genUID()
       ToSet(
         StreamFilter(ToStream(s1), x,
           contains(s2, Ref(x, t))))
     }
 
-    registerIR2("difference", TSet(tv("T")), TSet(tv("T")), TSet(tv("T"))) { (_, s1, s2) =>
+    registerIR2("difference", TSet(tv("T")), TSet(tv("T")), TSet(tv("T"))) { (_, s1, s2, _) =>
       val t = s1.typ.asInstanceOf[TSet].elementType
       val x = genUID()
       ToSet(
         StreamFilter(ToStream(s1), x,
           ApplyUnaryPrimOp(Bang(), contains(s2, Ref(x, t)))))
     }
 
-    registerIR2("isSubset", TSet(tv("T")), TSet(tv("T")), TBoolean) { (_, s, w) =>
+    registerIR2("isSubset", TSet(tv("T")), TSet(tv("T")), TBoolean) { (_, s, w, _) =>
       val t = s.typ.asInstanceOf[TSet].elementType
       val a = genUID()
       val x = genUID()
       StreamFold(ToStream(s), True(), a, x,
         // FIXME short circuit
-        ApplySpecial("land", FastSeq(), FastSeq(Ref(a, TBoolean), contains(w, Ref(x, t))), TBoolean))
+        ApplySpecial("land", FastSeq(), FastSeq(Ref(a, TBoolean), contains(w, Ref(x, t))), TBoolean, ErrorIDs.NO_ERROR))
```

Comment:
You have an error id available here. Try adding a test for this error to make sure it's a `HailUserError`